pyairtable/api/api.py

```diff
@@ -1,5 +1,6 @@
 from typing import List
-from .abstract import ApiAbstract
+
+from .abstract import ApiAbstract, DEFAULT_MAX_RECORDS_PER_REQUEST
```

Comment:
Accepting my suggestions for keyword-only arguments means this import would be unnecessary
```suggestion

from .abstract import ApiAbstract
```